lib/components/PageHeader/Account/AccountButton.jsx

```diff
@@ -10,9 +11,14 @@ import { ThemedClipSpinner } from '../../Loading/ThemedClipSpinner'
 export function AccountButton(props) {
   const { openModal, className, t } = props
 
+  const usersAddress = useUsersAddress()
+
   const [transactions] = useAtom(transactionsAtom)
   const pendingTransactionsCount = transactions.filter((t) => !t.completed).length
 
+  const { provider } = useOnboard()
```

Comment:
If a user changes networks this will try to look up an ens name on that network, is that what we want?

Reply:
Nope, good point. I wasn't thinking about other networks, but just assumed it would fail and revert to `''` (or just show the address).

I like the idea of always using ETH Mainnet for ENS 👍